Week8 - homework SQL/SQLQuery_2.sql

```diff
@@ -0,0 +1,157 @@
+CREATE DATABASE ZooManagement;
+
+USE ZooManagement;
+
+CREATE TABLE TicketTypes
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Subscription VARCHAR(20),
+Price INT,
+Validity VARCHAR(20),
+);
+
+CREATE TABLE Visitors
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+LastName VARCHAR(50),
+FirstName VARCHAR(50),
+TicketId INT REFERENCES TicketTypes(Id),
```

Comment:
Another table Tickets would be necessary - you could store there, as an example, the date and time when the ticket was bought so you can do some reports about "what is the most visited day / time during the day". The Visitors would be connected to the Tickets and the Tickets to TicketTypes.